weeks.txt

```diff
@@ -0,0 +1 @@
+Wednesday on github
```

Comment:
Please take into account that it will be Thursday tomorrow.

Reply:
ohh thanks for the info